contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -136,19 +142,12 @@ void multisig::unapprove( name proposer, name proposal_name, permission_level le
    proposals proptable( get_self(), proposer.value );
    auto& prop = proptable.get( proposal_name.value, "proposal not found" );
 
-   if ( prop.earliest_exec_time.has_value() ) {
-      if ( prop.earliest_exec_time.value().has_value() ) {
-         auto table_op = [](auto&&, auto&&){};
-         if ( approvals_satisfy_trx_authorization(proposer, proposal_name, prop.packed_transaction, table_op) ) {
-            return;
-         } else {
-            auto prop_it = proptable.find( proposal_name.value );
-            proptable.modify( prop_it, proposer, [&]( auto& p ) {
+   if ( prop.earliest_exec_time->has_value() ) {
```

Comment:
This change has a similar problem as the one described earlier for the `approve` action. I expect it to instead be:

```cpp
if( prop.earliest_exec_time.has_value() ) { 
   if( prop.earliest_exec_time->has_value() ) {
      auto table_op = [](auto&&, auto&&){};
      if( !trx_is_authorized(get_approvals_and_adjust_table(get_self(), proposer, proposal_name, table_op), prop.packed_transaction) ) {
         ...
      }
   }
} else {
   transaction_header trx_header = get_trx_header(prop.packed_transaction.data(), prop.packed_transaction.size());
   check( trx_header.delay_sec.value == 0, "old proposals are not allowed to have non-zero `delay_sec`; cancel and retry" );
}
```

Reply:
https://github.com/EOSIO/eosio.contracts/blob/ee3cdb17f330aed66e54ef51aa839e4af7c104c6/contracts/eosio.msig/src/eosio.msig.cpp#L169-L181